.github/workflows/regressions.yml

```diff
@@ -0,0 +1,72 @@
+name: Regressions
+on: workflow_dispatch
+
+env:
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 1
+  PUPPETEER_EXECUTABLE_PATH: /usr/bin/google-chrome
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    name: Build W3C profile
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-12-${{ hashFiles('**/package-lock.json') }}
+      - run: npm ci
+      - run: npm run build:w3c
+      - uses: actions/upload-artifact@v2
+        with:
+          name: builds
+          path: |
+            builds/respec-w3c.js*
+            builds/respec-worker.js*
+            builds/respec-highlight.js*
+          retention-days: 3
+
+  test:
+    name: Diff ${{ matrix.source }}
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        source:
+          - https://w3c.github.io/manifest/
+          - https://w3c.github.io/payment-request/
+          - https://w3c.github.io/trace-context/
```

Comment:
maybe https://w3c.github.io/screen-wake-lock/ instead... at it's one that we control?

Reply:
You can probably add geolocation too and maybe push API... they might be good as they don't change very much.

Reply:
Not changing much is not a concern, as we're diffing the live spec with different respec versions.

Reply:
`trace-context` is good for testing as it makes extensive use of `data-include="file.md" data-include-format='markdown'`.

Reply:
true... though we really can't assure the quality of that document.

Reply:
It's important to observe usage in the wild than in controlled environment.

Reply:
Can't argue with that :)